pkg/network/protocols/http/shared_libraries_test.go

```diff
@@ -64,11 +71,66 @@ func TestSharedLibraryDetection(t *testing.T) {
 	assert.Equal(t, fpath, pathDetected)
 }
 
+func TestSharedLibraryDetectionWithRoot(t *testing.T) {
+	t.Skip("skip for the moment as some distro are not friendly with busybox package")
+
+	tdir := t.TempDir()
+	root := filepath.Join(tdir, "root")
+	err := os.MkdirAll(root, 0755)
+	require.NoError(t, err)
+
+	libpath := "/fooroot.so"
+
+	simulateOpenAt(root + libpath)
+	err = exec.Command("cp", "/usr/bin/busybox", root+"/ash").Run()
+	require.NoError(t, err)
+	err = exec.Command("cp", "/usr/bin/busybox", root+"/sleep").Run()
```

Comment:
that's not really "sleep", the name is super confusing and does not give the ability to understand the flow of the test

Reply:
It is sleep, busybox change his personality based on the `basename(args[0])` this really useful in initrd or embedded environment (see busybox --install creating symlinks)
I remove the /sleep binary as it's not used directly, but sleep will be called from builtin ash command

Reply:
update: sleep need to be present in the path